test/eth/TransactionObject.spec.js

```diff
@@ -96,7 +96,11 @@ describe('normal web service behavior', () => {
 
     test(
       'generic error',
-      testErrorHandling(() => mkr.transfer(testAddress, '2000000'), /reverted/)
+      testErrorHandling(
+        () => mkr.transfer(testAddress, '2000000'),
+        /reverted/,
+        false
```

Comment:
why was this added?

Reply:
I think I can actually remove it now. Initially, web3's gas estimation function would go ahead and revert the transaction before it actually tried to send it, which meant that it was never pending. It caused unpredictable behavior in other error handling scenarios though, so I put `estimateGas` in a try/catch (and assigned the fallback gas limit in the catch) so transactions now fail where the tx manager expects them to.